src/website/app/demos/ThemeProvider/bestPractices.js

```diff
@@ -0,0 +1,49 @@
+export default [
+  {
+    type: 'do',
+    title: 'override theme variables at the top level',
+    example: `
+\`\`\`
+import { mineralTheme } from 'mineral-ui/utils';
+
+<ThemeProvider theme={{
+  ...mineralTheme,
+  color_primary: 'rebeccapurple'
+}}>
+  <App />
+</ThemeProvider>
+\`\`\`
+`,
+    description: `Try to keep theme overrides all in one place, as they will be easier to change later.`
+  },
+  {
+    type: 'do',
+    title: 'use a second or third themeprovider to style different sections',
+    example: `
+\`\`\`
+<ThemeProvider>
+  <div>The Main Part of your app</div>
+  <ThemeProvider theme={{
+    color_primary: 'limegreen'
+  }}>
+    <div>The admin section</div>
+  </ThemeProvider
+</ThemeProvider>
+\`\`\`
+`,
+    description: `You might want to style different sections of your app with different themes,
+    such as admin consoles, or a preferences section to create a visual separation.`
```

Comment:
I think this would be clearer without "or a preferences section". Otherwise, it needs reworded.

Reply:
addressed